base/applications/mspaint/winproc.cpp

```diff
@@ -587,6 +589,8 @@ LRESULT CMainWindow::OnSysColorChange(UINT nMsg, WPARAM wParam, LPARAM lParam, B
 
 LRESULT CMainWindow::OnCommand(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bHandled)
 {
+    CWaitCursor waitCursor;
```

Comment:
Why setting the cursor to wait mode while treating commands? It should be set to wait only for (very) time-consuming actions. For example, only in the rotate-mirror and skew actions (where you call the KeepWait), and just before calling the IsBlackAndWhite function (and restoring it afterwards), etc.

Reply:
Two points fixed. Thanks.